src/providers/MeetingProvider/docs/MeetingManager.stories.mdx

```diff
@@ -177,17 +238,23 @@ import { MeetingProvider } from 'amazon-chime-sdk-component-library-react';
 
 const MeetingForm = () => {
   const meetingManager = useMeetingManager();
-  const meetingSessionConfiguration = new DefaultMeetingSessionConfiguration(...);
+  
+  const response = await fetch('/my-meetings-endpoint');
+  const data = await response.json();
+  
+  const meetingSessionConfiguration = new MeetingSessionConfiguration(data.Meeting, data.Attendee);
+  const logger = new ConsoleLogger('SDK', LogLevel.WARN);
   const eventController = new DefaultEventController(
-        meetingSessionConfiguration, 
-        meetingManager.createLogger(meetingSessionConfiguration), 
-        new NoOpEventReporter()
-      );
+    meetingSessionConfiguration, 
+    logger, 
+    new NoOpEventReporter()
+  );
+
   const handleJoinMeeting = async (event) => {
-    await meetingManager.join({
+    await meetingManager.join(
       meetingSessionConfiguration,
-      eventController: eventController
-    });
+      options,
```

Comment:
Could you remove this comma as this is a function call hence wont be correct to have it here.